src/plugins/upf/upf_pfcp_api.c

```diff
@@ -1034,7 +828,7 @@ handle_update_pdr (upf_session_t * sx, pfcp_update_pdr_t * update_pdr,
 	  unformat_init_vector (&input, sdf->flow);
 	  vec_add2 (update->pdi.acl, acl, 1);
 
-	  if (!unformat_ipfilter (&input, acl))
+	  if (!unformat (&input, "%U", unformat_ipfilter, acl))
```

Comment:
I this change really needed?

Reply:
`unformat_ipfilter` now has a signature suitable for `unformat()`, namely `static uword
unformat_ipfilter_address_port (unformat_input_t * i, va_list * args)`, because it is used for CLI parsing via `unformat()`: https://github.com/travelping/vpp/blob/e2685c35b696108fda6a8b8729de59124a970999/src/plugins/upf/upf_adf.c#L737
So, it's now harder to use it directly.